Extensions/old_stats.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Old Stats **//
-//* VERSION 0.2.2 **//
+//* VERSION 0.3.0 **//
```

Comment:
Given that it probably shouldn't have been pre-1.0.0 for this long, getting it _out_ of pre-1.0.0 status would be ideal.

But I agree, nitpicking versions on a soon-to-be removed extension _is_ a bit pointless.